doc/glossary.rst

```diff
@@ -57,6 +57,8 @@ Common abbreviations in NEST
    virtual process
 
 Physical units and variable names used for NEST parameters
+:orphan:
+
```

Comment:
This should be moved to the beginning of the file. The way it is now breaks the formatting of the headline